charts/snyk-broker/templates/broker_deployment.yaml

```
@@ -58,7 +58,7 @@ spec:
             httpGet:
               port: {{ .Values.deployment.container.containerPort }}
               path: {{ .Values.brokerLivenessProbe.path }}
-              {{- if and (.Values.httpsCert) (.Values.httpsKey) }}
+              {{- if .Values.enableBrokerServerOverHttps }}
```

This changes existing behaviour - we should maintain backwards compatibility:

```suggestion
              {{- if or ( and (.Values.httpsCert) (.Values.httpsKey) ) ( .Values.enableBrokerServerOverHttps ) ) }}
```

---

@soniqua I see your point but my thought process here was keep a single flag to choose between https vs http. If httpsEnabled flag is set httpsCert and httpsKey are mandatory.

---

ahh from that lens, yes extra value will be required. I see your point now, I ll add the double conditional. But for the new hybrid platform chart we should keep a single flag to enable/disable something.